resource/api/swagger.yml

```diff
@@ -222,6 +222,30 @@ paths:
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: '#/definitions/Error'
+  /websiteCreator/state/{contractAddress}:
+    get:
+      operationId: websiteCreatorGetterStatus
+      parameters:
+        - in: path
+          name: contractAddress
+          type: string
+          required: true
+          description: Website creator contract's address
```

Comment:
@SlnPons I don't think that this descirption is sufficient for an external developer to understand this API endpoint. wdyt?

Reply:
I've to say I don't understand it myself. Tbh, there is an important effort to make and time to invest on the documentation and clarity of the fct ==> should be part of DoD and technical standards.

Reply:
It's Updated, wdyt ?

Reply:
@SlnPons this question is for you :-)